bot/seasons/evergreen/bookmark.py

```diff
@@ -25,6 +25,18 @@ def __init__(self, bot: commands.Bot):
     ) -> None:
         """Send the author a link to `target_message` via DMs."""
         log.info(f"{ctx.author} bookmarked {target_message.jump_url} with title '{title}'")
+        permissions = ctx.author.permissions_in(target_message.channel)
+        if not permissions.read_messages:
+            embed = discord.Embed(
+                title=random.choice(ERROR_REPLIES),
+                color=Colours.soft_red,
+                description="You don't have permission to see what's in the channel so you can't "
+                            "bookmark a message from it."
+            )
+            embed.set_author(name=ctx.author.name, icon_url=ctx.author.avatar_url)

```

Comment:
Are you sure that you want the error embed to be authored by the user who requested the bm?
![image](https://user-images.githubusercontent.com/44734341/73122884-0e873480-3f8a-11ea-9556-57dafc00bbfa.png)

Reply:
It's make you fell that error is for you only 
but at same time doesn't match with other error style . so gonna remove that.